cli/run_util.go

```diff
@@ -912,33 +704,32 @@ func activateAndRunSteps(
 
 		//
 		// Run step
-		bitrise.PrintRunningStepHeader(stepInfoPtr, mergedStep, idx)
```

Comment:
Moved to [BuildRunResultRegisterer.RegisterStepRunResults](https://github.com/bitrise-io/bitrise/pull/816/files#diff-c2a40f516c80b858ce25d8a9cfe7443c6ae26667bb972023705765d44323f71cR80)